<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpForm</class>
 <widget class="QWidget" name="helpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <widget class="QTextBrowser" name="helpTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
